xinference/model/llm/llm_family.json

```diff
@@ -617,5 +617,125 @@
       }
     ],
     "prompt_style": null
+  },
+  {
+    "version": 1,
+    "model_name": "falcon",
```

Comment:
only need one `falcon` model_family.
`model_specs` contains 7b and 40b configs.
you can refer to other model_family like `vicuna`